dask/core.py

```diff
@@ -194,3 +195,31 @@ def reverse_dict(d):
             result[val].add(k)
     return result
 
+
+def cull(dsk, keys):
+    """ Return new dask with only the tasks required to calculate keys.
+
+    In other words, remove unnecessary tasks from dask.
+    ``keys`` may be a single key or list of keys.
+
+    Exmaple
+    -------
+
+    >>> d = {'x': 1, 'y': (inc, 'x'), 'out': (add, 'x', 10)}
+    >>> cull(d, 'out')  # doctest: +SKIP
+    {'x': 1, 'out': (add, 'x', 10)}
+    """
+    if not isinstance(keys, list):
```

Comment:
I believe there is already a precedence to only use lists to specify multiple keys.  After all, a tuple is hashable and may be used as a key.